pennylane/ops/identity.py

```diff
@@ -46,8 +45,18 @@ def label(self, decimals=None, base_label=None):
         return base_label or "I"
 
     @classmethod
-    def _eigvals(cls, *params):
-        return cls.eigvals
+    def compute_eigvals(cls):  # pylint: disable=unused-argument,arguments-differ
```

Comment:
shouldn't this be updated to a static method?

Reply:
Oh I forgot to leave a comment! We need `cls`, else we cannot access `compute_matrix`! Or do you have another idea?

Reply:
oh!!!! Ah 🙁

Reply:
We can decide whether or not to make _all_ `compute_` class methods, it will be easy to change when everything is on the dev branch!